distribution/docs/src/main/resources/content/_configuring/uploading-and-editing.adoc

```diff
@@ -0,0 +1,35 @@
+:title: Configuring User Access to Ingest and Metadata for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 02
+:summary: Configuring user ability to upload products and edit existing metadata in ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} lets the administrator control user access to ingest and metadata. The administrator
+can show or hide the uploader, letting them control whether users can ingest products. They
+can also choose whether or not users can edit existing metadata. By default, the uploader is
+available to users and editing is allowed.
+
+.Configuring The Uploader
+Choose to hide or show the uploader.
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-ui}* application.
+. Select the *Configuration* tab.
+. Select Catalog UI Search.
+. Scroll down to the "Show Uploader" option.
+. Click the checkbox.
```

Comment:
✏️ We can probably combine this into a single step and say "Select *Show Uploader*."